pyproject.toml

```diff
@@ -0,0 +1,64 @@
+[build-system]
+requires = [
+    "hatchling>=0.25.1",
+]
+build-backend = "hatchling.build"
+
+[project]
+name = "colorama"
+description = "Cross-platform colored terminal text."
+readme = "README.rst"
+license = "BSD-3-Clause"
+requires-python = ">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, !=3.5.*, !=3.6.*"
+authors = [
+    { name = "Jonathan Hartley", email = "tartley@tartley.com" },
+]
+keywords = [
+    "ansi",
+    "color",
+    "colour",
+    "crossplatform",
+    "terminal",
+    "text",
+    "windows",
+    "xplatform",
+]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Environment :: Console",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: BSD License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 2",
+    "Programming Language :: Python :: 2.7",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: Implementation :: CPython",
+    "Programming Language :: Python :: Implementation :: PyPy",
+    "Topic :: Terminals",
+]
+dynamic = [
+    "version",
+]
+
+[project.urls]
+Homepage = "https://github.com/tartley/colorama"
+
+[tool.hatch.version]
+path = "colorama/__init__.py"
+
+[tool.hatch.build.targets.sdist]
+include = [
+    "/colorama",
+    "/demos",
+    "/CHANGELOG.rst",
+]
+
+[tool.hatch.build.targets.wheel]
+include = [
+    "/colorama/*",
+]
```

Comment:
Why don't the wheel and sdist have the same files included?

Reply:
Tests weren't being shipped in the wheel already (rightly so imo) https://pypi.org/project/colorama/#files :

![w](https://user-images.githubusercontent.com/9677399/174487578-46eada23-0c17-4355-96dd-21d823ad29b1.PNG)